### ISSUE TITLE
On startup, Analogy displays a message box with an error

### ISSUE THREAD

**Issue by dagophil** [Analogy-LogViewer/Analogy.LogViewer#1942]
## Describe the bug
When I start the application, it displays a message box with `InvalidOperationException`.

## ℹ Computer information

 - OS: Windows 10 Enterprise 22H2 64 bit
 - Analogy Version: 6.0.1.0 .NET 8
 - Data Provider used: Only Serilog, all other disabled.

## 📝 Provide detailed reproduction steps (if any)

1. Just start Analogy. I also started a fresh downloaded version and the same error occurs.

### ✔️ Expected result

Analogy starts without error.

### ❌ Actual result

Analogy displays a message box with an error.

## 📷 Screenshots

![image](https://github.com/Analogy-LogViewer/Analogy.LogViewer/assets/6207208/7548d397-fc5f-41ed-876a-acde8ccce480)

**Comment by dagophil** [comment]
Judging from the code that is mentioned in the callstack Analogy/Managers/UpdateManager.cs line 162 it seems as if the update manager cannot find any release. One possible error source could be that the request is blocked by the firewall on my machine. Another possible source could be that I am behind a proxy.

While an error message can be okay in that case, it would be nice to have it a little bit more readable. For example, the message box could display something like _Error: Failed to load version information from the server_

**Comment by LiorBanai** [comment]
@dagophil  I agree with your analysis and suggestion :)
Will implement it :)

Thanks!

**Comment by dagophil** [comment]
@LiorBanai Thank you :)
